# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../kvm/00-no-bios-date-host.patch
# Copyright (C) 2010 - 2012 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

A BIOS version changing with every date and host is too much.
SuSE patches it away as well, ...

  - Rene Rebe <rene@exactcode.de>

--- qemu-kvm-1.0.1/roms/seabios/tools/buildversion.sh.vanilla	2012-05-15 16:06:35.462849893 +0000
+++ qemu-kvm-1.0.1/roms/seabios/tools/buildversion.sh	2012-05-15 16:06:41.990850030 +0000
@@ -12,6 +12,7 @@
     VERSION="?"
 fi
 VERSION="${VERSION}-`date +"%Y%m%d_%H%M%S"`-`hostname`"
+VERSION=1.7.0
 echo "Version: ${VERSION}"
 
 # Build header file
--- qemu-kvm-1.0.1/roms/seabios-mac/tools/buildversion.sh.vanilla	2012-05-15 16:06:35.462849893 +0000
+++ qemu-kvm-1.0.1/roms/seabios-mac/tools/buildversion.sh	2012-05-15 16:06:41.990850030 +0000
@@ -12,6 +12,7 @@
     VERSION="?"
 fi
 VERSION="${VERSION}-`date +"%Y%m%d_%H%M%S"`-`hostname`"
+VERSION=1.7.0
 echo "Version: ${VERSION}"
 
 # Build header file
